torch/csrc/jit/mobile/register_mobile_ops.cpp

```diff
@@ -132,6 +133,37 @@ void warn_kernel(const c10::OperatorHandle& op, Stack* stack) {
   pop(*stack);
 }
 
+void tupleunpack_kernel(const c10::OperatorHandle& op, Stack* stack) {
+  auto tuple = pop(*stack).toTuple();
+  stack->insert(
+     stack->end(),
+     tuple->elements().begin(),
+     tuple->elements().end());
+}
+
+void format_kernel(const c10::OperatorHandle& op, Stack* stack) {
```

Comment:
This looks duplicated from full jit.  Any way to reuse code?

Reply:
This can just call formatFunc from vararg_functions.h

Reply:
vararg_functions.cpp may not be included in the BUCK target for internal build. Since this file will be removed soon, I may just land it for now to quickly resolve the internal test issue, and remove them together with the file.